controllers/profiles/reconciler_dev.go

```diff
@@ -311,11 +317,75 @@ func mountDevConfigMapsMutateVisitor(flowDefCM, propsCM *v1.ConfigMap) mutateVis
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = make([]v1.VolumeMount, 0)
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = volumeMounts
 
+			for k, val := range workflow.Labels {
+				if strings.HasPrefix(k, extResPrefix) {
+					cm, err := getGenericConfigMap(val, workflow.Namespace, support)
+					if err != nil {
+						log.Errorf(err, "ConfigMap not found %s", val)
+					} else {
+						deployment := object.(*appsv1.Deployment)
+						for k, _ := range cm.Data {
+							if !isContained(k, deployment.Spec.Template.Spec.Volumes) {
+
+								deployment.Spec.Template.Spec.Volumes = append(deployment.Spec.Template.Spec.Volumes,
+									v1.Volume{
+										Name: configMapExternalPrefix,
+										VolumeSource: v1.VolumeSource{
+											ConfigMap: &v1.ConfigMapVolumeSource{
+												LocalObjectReference: v1.LocalObjectReference{Name: val},
+											},
+										},
+									})
+
+								deployment.Spec.Template.Spec.Containers[0].VolumeMounts = append(deployment.Spec.Template.Spec.Containers[0].VolumeMounts,
+									v1.VolumeMount{
+										Name:      configMapExternalPrefix,
+										ReadOnly:  true,
+										MountPath: quarkusDevConfigMountPath,
```

Comment:
Each resource should have its own mount path.

Reply:
Aren't all under src/main/resources ?

Reply:
No, they have exclusive paths, for camel, we must mount within `src/main/resources/routes`.

Reply:
@ricardozanini which are the mount for OpenAPI and AsyncAPI  ?

Reply:
`src/main/resources/specs/`

Add to the document, let's exhaust the discussion there.

Reply:
btw, try a few examples with the builder image and mounting a local volume to see if it works